gnuradio-runtime/lib/terminate_handler.cc

```diff
@@ -0,0 +1,147 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2018 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+#include <pmt/pmt.h>
+#include <boost/thread.hpp>
+#include <exception>
+#include <iostream>
+#include <regex>
+#include <stdexcept>
+
+#include <cxxabi.h>
```

Comment:
Where are MSVC/clang-cl supposed to pick up this file from?
Surely it should be:
```diff
--- a/terminate_handler.cc 2020-12-19 16:41:48
+++ b/terminate_handler.cc 2020-12-19 17:18:19
@@ -14,7 +14,9 @@
 #include <regex>
 #include <stdexcept>

+#ifdef __GNUC__
 #include <cxxabi.h>
+#endif
```